arangopipe/arangopipe/docker-compose.yml

```diff
@@ -0,0 +1,17 @@
+version: '3'
+services:
+  arangodb:
+    environment:
+      - ARANGO_NO_AUTH=1
+    image: "arangodb:3.4"
```

Comment:
Why do we still need docker compose with the single image?

Reply:
Ok, I can remove the docker compose artifacts.